quarkus main build fails on windows

### Describe the bug

mvnw -Dquickly -e

### Expected behavior

quarkus main should always at least compile.

### Actual behavior

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) on project quarkus-kubernetes-service-binding: Execution default-testCompile of goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile failed: Internal error in Develocity M
aven extension: goal execution: Failed to load cache entry for org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) @ io.quarkus:quarkus-kubernetes-service-binding:jar:999-SNAPSHOT: Could not load from remote cache: Cannot create directory 'C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\.hidden-symlink-dir'. -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) on project quarkus-kubernetes-service-binding: Execution default-testCompile of goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile failed: Internal error in Develocity Maven extension: goal execution
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:333)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default-testCompile of goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile failed: Internal error in Develocity Maven extension: goal execution
    at com.gradle.maven.scan.extension.internal.d.b.executeMojo (SourceFile:123)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: com.gradle.maven.common.logging.DevelocityException: Internal error in Develocity Maven extension: goal execution
    at com.gradle.maven.common.logging.DevelocityException.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.m.a (SourceFile:106)
    at com.gradle.maven.cache.extension.d.m.a (SourceFile:90)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:76)
    at jdk.proxy34.$Proxy141.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.b.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: java.lang.RuntimeException: Failed to load cache entry for org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) @ io.quarkus:quarkus-kubernetes-service-binding:jar:999-SNAPSHOT
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:87)
    at com.gradle.maven.cache.extension.d.b.b (SourceFile:80)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:50)
    at com.gradle.maven.cache.extension.d.g.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.o.a (SourceFile:18)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.c.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.h$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.h.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.m.a (SourceFile:82)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:76)
    at jdk.proxy34.$Proxy141.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.b.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: org.gradle.caching.internal.controller.BuildCacheOperationException: Could not load from remote cache: Cannot create directory 'C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\.hidden-symlink-dir'.
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.lambda$loadRemoteAndStoreResultLocally$2 (SourceFile:140)
    at org.gradle.caching.local.internal.DefaultBuildCacheTempFileStore.withTempFile (SourceFile:37)
    at org.gradle.caching.local.internal.DirectoryBuildCache.lambda$withTempFile$5 (SourceFile:175)
    at org.gradle.cache.internal.DefaultCacheCoordinator.lambda$toSupplier$2 (SourceFile:228)
    at org.gradle.cache.internal.LockOnDemandCrossProcessCacheAccess.withFileLock (SourceFile:90)
    at org.gradle.cache.internal.DefaultCacheCoordinator.withFileLock (SourceFile:218)
    at org.gradle.cache.internal.DefaultPersistentDirectoryStore.withFileLock (SourceFile:152)
    at org.gradle.cache.internal.DefaultCacheFactory$ReferenceTrackingCache.withFileLock (SourceFile:208)
    at org.gradle.caching.local.internal.DirectoryBuildCache.withTempFile (SourceFile:175)
    at org.gradle.caching.local.internal.DirectoryBuildCacheService.withTempFile (SourceFile:65)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.loadRemoteAndStoreResultLocally (SourceFile:135)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.load (SourceFile:119)
    at com.gradle.maven.cache.extension.b.c.a (SourceFile:32)
    at com.gradle.maven.cache.extension.b.f$a.a (SourceFile:292)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:85)
    at com.gradle.maven.cache.extension.d.b.b (SourceFile:80)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:50)
    at com.gradle.maven.cache.extension.d.g.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.o.a (SourceFile:18)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.c.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.h$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.h.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.m.a (SourceFile:82)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:76)
    at jdk.proxy34.$Proxy141.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.b.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: org.gradle.internal.operations.BuildOperationInvocationException: Cannot create directory 'C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\.hidden-symlink-dir'.
    at org.gradle.internal.operations.DefaultBuildOperationRunner.throwAsBuildOperationInvocationException (SourceFile:197)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.access$100 (SourceFile:24)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:75)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.call (SourceFile:53)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor.unpack (SourceFile:189)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.lambda$loadRemoteAndStoreResultLocally$1 (SourceFile:138)
    at org.gradle.caching.internal.controller.service.BaseRemoteBuildCacheServiceHandle.maybeUnpack (SourceFile:118)
    at org.gradle.caching.internal.controller.service.BaseRemoteBuildCacheServiceHandle.maybeLoad (SourceFile:105)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.lambda$loadRemoteAndStoreResultLocally$2 (SourceFile:138)
    at org.gradle.caching.local.internal.DefaultBuildCacheTempFileStore.withTempFile (SourceFile:37)
    at org.gradle.caching.local.internal.DirectoryBuildCache.lambda$withTempFile$5 (SourceFile:175)
    at org.gradle.cache.internal.DefaultCacheCoordinator.lambda$toSupplier$2 (SourceFile:228)
    at org.gradle.cache.internal.LockOnDemandCrossProcessCacheAccess.withFileLock (SourceFile:90)
    at org.gradle.cache.internal.DefaultCacheCoordinator.withFileLock (SourceFile:218)
    at org.gradle.cache.internal.DefaultPersistentDirectoryStore.withFileLock (SourceFile:152)
    at org.gradle.cache.internal.DefaultCacheFactory$ReferenceTrackingCache.withFileLock (SourceFile:208)
    at org.gradle.caching.local.internal.DirectoryBuildCache.withTempFile (SourceFile:175)
    at org.gradle.caching.local.internal.DirectoryBuildCacheService.withTempFile (SourceFile:65)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.loadRemoteAndStoreResultLocally (SourceFile:135)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.load (SourceFile:119)
    at com.gradle.maven.cache.extension.b.c.a (SourceFile:32)
    at com.gradle.maven.cache.extension.b.f$a.a (SourceFile:292)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:85)
    at com.gradle.maven.cache.extension.d.b.b (SourceFile:80)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:50)
    at com.gradle.maven.cache.extension.d.g.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.o.a (SourceFile:18)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.c.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.h$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.h.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.m.a (SourceFile:82)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:76)
    at jdk.proxy34.$Proxy141.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.b.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: java.io.IOException: Cannot create directory 'C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\.hidden-symlink-dir'.
    at com.gradle.maven.extension.internal.dep.org.apache.commons.io.FileUtils.mkdirs (SourceFile:2373)
    at com.gradle.maven.extension.internal.dep.org.apache.commons.io.FileUtils.forceMkdir (SourceFile:1431)
    at org.gradle.caching.internal.packaging.impl.TarBuildCacheEntryPacker.unpackDirectoryTree (SourceFile:298)
    at org.gradle.caching.internal.packaging.impl.TarBuildCacheEntryPacker.unpackTree (SourceFile:266)
    at org.gradle.caching.internal.packaging.impl.TarBuildCacheEntryPacker.unpack (SourceFile:210)
    at org.gradle.caching.internal.packaging.impl.TarBuildCacheEntryPacker.unpack (SourceFile:172)
    at org.gradle.caching.internal.packaging.impl.GZipBuildCacheEntryPacker.unpack (SourceFile:49)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor.doUnpack (SourceFile:210)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor.access$000 (SourceFile:174)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor$1.call (SourceFile:194)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor$1.call (SourceFile:189)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute (SourceFile:209)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute (SourceFile:204)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.call (SourceFile:53)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor.unpack (SourceFile:189)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.lambda$loadRemoteAndStoreResultLocally$1 (SourceFile:138)
    at org.gradle.caching.internal.controller.service.BaseRemoteBuildCacheServiceHandle.maybeUnpack (SourceFile:118)
    at org.gradle.caching.internal.controller.service.BaseRemoteBuildCacheServiceHandle.maybeLoad (SourceFile:105)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.lambda$loadRemoteAndStoreResultLocally$2 (SourceFile:138)
    at org.gradle.caching.local.internal.DefaultBuildCacheTempFileStore.withTempFile (SourceFile:37)
    at org.gradle.caching.local.internal.DirectoryBuildCache.lambda$withTempFile$5 (SourceFile:175)
    at org.gradle.cache.internal.DefaultCacheCoordinator.lambda$toSupplier$2 (SourceFile:228)
    at org.gradle.cache.internal.LockOnDemandCrossProcessCacheAccess.withFileLock (SourceFile:90)
    at org.gradle.cache.internal.DefaultCacheCoordinator.withFileLock (SourceFile:218)
    at org.gradle.cache.internal.DefaultPersistentDirectoryStore.withFileLock (SourceFile:152)
    at org.gradle.cache.internal.DefaultCacheFactory$ReferenceTrackingCache.withFileLock (SourceFile:208)
    at org.gradle.caching.local.internal.DirectoryBuildCache.withTempFile (SourceFile:175)
    at org.gradle.caching.local.internal.DirectoryBuildCacheService.withTempFile (SourceFile:65)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.loadRemoteAndStoreResultLocally (SourceFile:135)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.load (SourceFile:119)
    at com.gradle.maven.cache.extension.b.c.a (SourceFile:32)
    at com.gradle.maven.cache.extension.b.f$a.a (SourceFile:292)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:85)
    at com.gradle.maven.cache.extension.d.b.b (SourceFile:80)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:50)
    at com.gradle.maven.cache.extension.d.g.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.o.a (SourceFile:18)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.c.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.h$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.h.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.m.a (SourceFile:82)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:76)
    at jdk.proxy34.$Proxy141.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.b.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
[ERROR]
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :quarkus-kubernetes-service-binding
```

### How to Reproduce?

_No response_

### Output of `uname -a` or `ver`

Microsoft Windows [Version 10.0.22621.4317]

### Output of `java -version`

openjdk 17.0.1 2021-10-19 OpenJDK Runtime Environment Temurin-17.0.1+12 (build 17.0.1+12) OpenJDK 64-Bit Server VM Temurin-17.0.1+12 (build 17.0.1+12, mixed mode, sharing)

### Quarkus version or git rev

6aafee46bad5d32f5e2740c596baed37f330f09c

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.9.9 (8e8579a9e76f7d015ee5ec7bfcdc97d260186937)
Maven home: C:\Users\Martin\.m2\wrapper\dists\apache-maven-3.9.9-bin\33b4b2b4\apache-maven-3.9.9
Java version: 17.0.1, vendor: Eclipse Adoptium, runtime: C:\tools\java\17-temurin
Default locale: de_DE, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"

### Additional information

_No response_

ok, I followed the guide over at [1] to enable symlinks in my windows, and to allow my git clone to use symlinks.
Required me to enable developer mode. git config core.symlinks true. delete extensions/kubernetes-service-binding/runtime/src/test/resources/k8s/test-k8s/.hidden-symlink-dir. and `it restore extensions/kubernetes-service-binding/runtime/src/test/resources/k8s/test-k8s/.hidden-symlink-dir


Hower, on mvnw -quickly I not get this one:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-resources-plugin:3.3.1:testResources (default-testResources) on project quarkus-kuber
netes-service-binding: copying C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\src\test\resources\k8s\test-k8s\p
rovider to C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\provider failed with
 FileSystemException: C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\src\test\resources\k8s\test-k8s\provider -
> C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\provider: Dem Client fehlt ein erforderliches Recht -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-resources-plugin:3.3.1:testResources 
(default-testResources) on project quarkus-kubernetes-service-binding: copying C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-b
inding\runtime\src\test\resources\k8s\test-k8s\provider to C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\provider failed with FileSystemException
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:333)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: org.apache.maven.plugin.MojoExecutionException: copying C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runti
me\src\test\resources\k8s\test-k8s\provider to C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\provider failed with FileSystemException
    at org.apache.maven.plugins.resources.ResourcesMojo.execute (ResourcesMojo.java:347)
    at org.apache.maven.plugins.resources.TestResourcesMojo.execute (TestResourcesMojo.java:70)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at com.gradle.maven.cache.extension.d.n.a (SourceFile:53)
    at com.gradle.maven.cache.extension.d.d.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.q.a (SourceFile:23)
    at com.gradle.maven.cache.extension.d.j.a (SourceFile:28)
    at com.gradle.maven.cache.extension.d.p.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.b.c (SourceFile:118)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:62)
    at com.gradle.maven.cache.extension.d.g.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.o.a (SourceFile:18)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.c.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.h$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.h.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.m.a (SourceFile:82)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:76)
    at jdk.proxy9.$Proxy93.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.b.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: org.apache.maven.shared.filtering.MavenFilteringException: copying C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-bi
nding\runtime\src\test\resources\k8s\test-k8s\provider to C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\provider failed with FileSystemException
    at org.apache.maven.shared.filtering.DefaultMavenFileFilter.copyFile (DefaultMavenFileFilter.java:116)
    at org.apache.maven.shared.filtering.DefaultMavenResourcesFiltering.filterResources (DefaultMavenResourcesFiltering.java:268)
    at org.apache.maven.plugins.resources.ResourcesMojo.execute (ResourcesMojo.java:343)
    at org.apache.maven.plugins.resources.TestResourcesMojo.execute (TestResourcesMojo.java:70)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at com.gradle.maven.cache.extension.d.n.a (SourceFile:53)
    at com.gradle.maven.cache.extension.d.d.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.q.a (SourceFile:23)
    at com.gradle.maven.cache.extension.d.j.a (SourceFile:28)
    at com.gradle.maven.cache.extension.d.p.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.b.c (SourceFile:118)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:62)
    at com.gradle.maven.cache.extension.d.g.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.o.a (SourceFile:18)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.c.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.h$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.h.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.m.a (SourceFile:82)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:76)
    at jdk.proxy9.$Proxy93.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.b.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: java.nio.file.FileSystemException: C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\src\test\resources
\k8s\test-k8s\provider -> C:\Users\Martin\IdeaProjects\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\provider: Dem Client fehlt ein erforderliches Recht
    at sun.nio.fs.WindowsException.translateToIOException (WindowsException.java:92)
    at sun.nio.fs.WindowsException.rethrowAsIOException (WindowsException.java:103)
    at sun.nio.fs.WindowsFileCopy.copy (WindowsFileCopy.java:209)
    at sun.nio.fs.WindowsFileSystemProvider.copy (WindowsFileSystemProvider.java:284)
    at java.nio.file.Files.copy (Files.java:1305)
    at org.apache.maven.shared.filtering.FilteringUtils.copyFile (FilteringUtils.java:318)
    at org.apache.maven.shared.filtering.DefaultMavenFileFilter.copyFile (DefaultMavenFileFilter.java:109)
    at org.apache.maven.shared.filtering.DefaultMavenResourcesFiltering.filterResources (DefaultMavenResourcesFiltering.java:268)
    at org.apache.maven.plugins.resources.ResourcesMojo.execute (ResourcesMojo.java:343)
    at org.apache.maven.plugins.resources.TestResourcesMojo.execute (TestResourcesMojo.java:70)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at com.gradle.maven.cache.extension.d.n.a (SourceFile:53)
    at com.gradle.maven.cache.extension.d.d.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.q.a (SourceFile:23)
    at com.gradle.maven.cache.extension.d.j.a (SourceFile:28)
    at com.gradle.maven.cache.extension.d.p.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.b.c (SourceFile:118)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:62)
    at com.gradle.maven.cache.extension.d.g.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.o.a (SourceFile:18)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:46)
    at com.gradle.maven.cache.extension.d.c.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.h$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.h.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.m.a (SourceFile:82)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:76)
    at jdk.proxy9.$Proxy93.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.b.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
[ERROR]
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -252,7 +252,6 @@ If you have not done so on this machine, you need to:
   * Windows:
     * enable longpaths: `git config --global core.longpaths true`
     * avoid CRLF breaks: `git config --global core.autocrlf false`
-    * enable symlinks: `git config --global core.symlinks true`
 * Install Java SDK 17+ (OpenJDK recommended)
 * Install [GraalVM](https://quarkus.io/guides/building-native-image)
 * Install platform C developer tools:

--- a/extensions/kubernetes-service-binding/runtime/pom.xml
+++ b/extensions/kubernetes-service-binding/runtime/pom.xml
@@ -67,5 +67,55 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>disable-test-compile-on-windows</id>
+            <activation>
+                <os>
+                    <family>windows</family>
+                </os>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-compiler-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>default-testCompile</id>
+                                <phase>test-compile</phase>
+                                <goals>
+                                    <goal>testCompile</goal>
+                                </goals>
+                                <configuration>
+                                    <skip>true</skip>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>default-testResources</id>
+                                <phase>test-resources</phase>
+                                <goals>
+                                    <goal>testResources</goal>
+                                </goals>
+                                <configuration>
+                                    <skip>true</skip>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <skip>true</skip>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
 
 </project>

